tests/shim.php

```diff
@@ -245,7 +240,7 @@ function getOptionsPostgresql(): array
     function getOptionsSqlite(): array
     {
         return [
-            'dbname' => env('DATA_SQLITE_NAME'),
+            'dbname' => dirname(__DIR__ ) . "/" . env('DATA_SQLITE_NAME'),
```

Comment:
Should this be here or maybe send it to the output folder of the tests - I think `outputFolder()` ?

Reply:
That is the output folder. Codeception need a relative path for sqlite. And we need it absolute. Will work on naming a bit.